README.md

```diff
@@ -40,7 +40,15 @@ TODO
 
 Use with Up board
 -----------------
-TODO
+On the Up board, there are only a few points you need to check if you have issues with permissions (symptom is usually an error `Could not set necessary configuration of serial port` from the [SBUS bridge](https://github.com/uzh-rpg/rpg_quadrotor_control) )
```

Comment:
I would make this link more specific: -> https://github.com/uzh-rpg/rpg_quadrotor_control/tree/master/bridges/sbus_bridge